core/templates/core/index.html

```diff
@@ -3,5 +3,16 @@
 {% block title %}Strona Główna{% endblock %}
 
 {% block content %}
-    <p class="text-3xl font-bold underline text-red-600">Gwardia Czapli</p>
+    <div class="flex flex-col flex-nowrap items-center">
+        <h1 class="text-4xl font-bold underline text-red-600 mt-10 mb-16">Gwardia Czapli</h1>
```

Comment:
Remove this

Reply:
Will change h1 to Gwardia Hub and h2 to Project goal